src/test/java/ortus/boxlang/runtime/bifs/global/system/GetSystemSettingTest.java

```diff
@@ -46,7 +46,7 @@ public void setupEach() {
 	public void testGetSystemSetting() {
 		instance.executeSource(
 		    """
-		    result = getSystemSetting( "Java.VerSion" )
```

Comment:
System settings retrieval should not be case sensitive.  Please revert this change as, if it fails, then the BIF itself has a bug.

Reply:
reverted `result = getSystemSetting( "Java.VerSion" )`